### DISCUSSION TITLE
Is there any way to load/save OpenEXR image？

### DISCUSSION THREAD

**Discussion by ksgfk** [SixLabors/ImageSharp#1890] [Q&A] [answered]
It seems that no C# library supports loading and saving OpenEXR images...

**Comment by dlemstra** [comment, marked as answer]
The ImageSharp library currently has no support for OpenEXR images. But this C# library is one of the libraries that has support for reading and writing OpenEXR images: https://github.com/dlemstra/Magick.NET.

**Reply by ksgfk** [reply]
Thanks.I will try this library.
By the way, does ImageSharp have any plans to support OpenEXR？
